EIPS/eip-3664.md

```diff
@@ -0,0 +1,418 @@
+---
+eip: 3664
+title: Next generation metaverse NFT
+description: This is a standard for next generation metaverse NFTs, which extends [ERC-1155](./eip-1155.md)
```

Comment:
```suggestion
description: Standard for next-generation metaverse NFTs, which extend ERC-1155
```

Reply:
@axic do you know if links are okay in the description field? My guess is no, but not sure.

Reply:
Sorry I missed this. Probably not as we run the escaping on it. The same applies to `withdrawal-reason` where I saw it is not rendered properly.

I think it would be nice to render both, but for the sake of merging this I'd consider it not possible.

Reply:
@nimocat still need to get this fixed before merging.